Content.Shared/Body/Components/SharedBodyComponent.cs

```diff
@@ -474,51 +472,8 @@ public IEnumerable<(SharedBodyPartComponent part, IBodyPartProperty property)> G
             }
         }
 
-        private void CalculateSpeed()
```

Comment:
why has this just been removed

Reply:
![image](https://user-images.githubusercontent.com/31366439/130358384-2df882c1-b45f-4d09-932b-9832daadad77.png)

https://discord.com/channels/310555209753690112/310555209753690112/877189830558253077